rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2021-11-17
+nightly-2022-07-26
```

Comment:
I'd update to the same that was just used in `zkevm-circuits`. See: https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/705